### DISCUSSION TITLE
这样会不会不安全

### DISCUSSION THREAD

**Discussion by autisn330** [lsky-org/lsky-pro#392] [想法]
2.0版本的图床需要开发数据库权限
![image](https://user-images.githubusercontent.com/92194759/164142473-77888ad0-d69e-464a-ada0-e8c7ef985812.png)
，不然页面就会这样
![image](https://user-images.githubusercontent.com/92194759/164142559-8dad343f-493a-4a43-95bb-9e6305ed76ed.png)
，旧版本的图床权限这样限制
![image](https://user-images.githubusercontent.com/92194759/164142636-287dd9a2-89a0-435f-a480-31fb7b33722b.png)
，实际这样访问并使用时没有问题的
![image](https://user-images.githubusercontent.com/92194759/164142701-c7938e69-abde-4df3-b677-3885849b385f.png)
请问这样操作容易被攻击嘛

**Comment by 0xxb** [comment]
改回「本地服务器」吧，如果数据库还是连不上就把数据库连接地址换成 `localhost` 再试试，或者改成服务器的内网IP。

**Reply by autisn330** [reply]
好的，我去试试

**Comment by akatsukiro** [comment]
这应该是你自己安装的时候访问地址填写的问题，我想你应该填的服务器公网IP。
![image](https://user-images.githubusercontent.com/69096852/164144285-32e6210a-dd3d-4db9-be0e-17ef9aa6ee8d.png)

**Reply by autisn330** [reply]
对的，填写的是公网ip，改成内网IP吗？
